WalletLibrary/Submodules/VerifiableCredential-SDK-iOS/VCEntities/VCEntities/issuance/IssuanceResponseContainer.swift

```diff
@@ -18,7 +14,7 @@ struct IssuanceResponseContainer: ResponseContaining {
     var requestedIdTokenMap: RequestedIdTokenMap = [:]
     var requestedAccessTokenMap: RequestedAccessTokenMap = [:]
     var requestedSelfAttestedClaimMap: RequestedSelfAttestedClaimMap = [:]
-    var requestVCMap: RequestedVerifiableCredentialMap = []
+    var requestVCMap: [RequestedVerifiableCredentialMapping] = []
```

Comment:
Why does issuance change? Is it a common data model?

Reply:
It is a common data model that was changes but only needed the change in presentation.